MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -2065,10 +2064,11 @@ public Part getPartByNameAndDetails(String pnd) {
     }
 
     protected void loadPlanetTSVFile() {
-        FileDialogs.openPlanetsTsv(frame).ifPresent(tsvFile -> {
+        //TODO: fix this
```

Comment:
Hanging TODO.

Reply:
Honestly, this method can probably be deleted, along with whatever references it, as the entire SUCS cartography feature import will have to be re-written, for two reasons. One, they changed the format of their excel file around, and two, now we will have systems instead of planets.

Reply:
Ok, we are using all of their data anyway. I have removed the method and references in commit b6cee504e5a7142a7cf42eddbab62bba8b3fd496.